LocalSettings.php

```diff
@@ -4487,6 +4491,16 @@
 			'XMP' => false,
 		],
 	],
+
+// Email notifications on privileged actions configuration
```

Comment:
Indentation is off I think? Needs one additional tab on everything it seems. And removal of tab on a bracket.

Reply:
Thanks, will fix that when I change my email to Owen's after testing.